README.md

```diff
@@ -55,7 +55,7 @@ linters:
 
 ## Rules
 
-- [GitHub::Accessibility::AvoidBothDisabledAndAriaDisabledCounter](./docs/rules/accessibility/avoid-both-disabled-and-aria-disabled-counter.md)
+- [GitHub::Accessibility::AvoidBothDisabledAndAriaDisabledCounter](./docs/rules/accessibility/avoid-both-disabled-and-aria-disabled.md)
```

Comment:
I will remove `Counter` from names in the README right before we cut release, for alignment with current version.